app/src/main/res/drawable/card.xml

```diff
@@ -4,10 +4,10 @@
     <item>
         <shape>
             <padding
-                android:top="8dp"
-                android:right="8dp"
-                android:bottom="5dp"
-                android:left="8dp"/>
+                android:top="0dp"
```

Comment:
I think that's better without additional external padding, so there's more space for title and artists name.